Добавлен отчёт по курсовой работе по дисциплине ИОСУ

Lelbry opened the pull request.

Lelbry committed:
Добавлен отчёт по курсовой работе по дисциплине ИОСУ

MarinaPaley reviewed (changes requested):
1. Шрифт текста должен быть с 1.5.пт междустрочным интервалом.
2. Должен быть пункт проверки с помощью нормализации.
3. Добавьте еще представление с внутренним соединением
4. И процедуру добавления данных, чтобы не руками данные добавлять
5. Вывод. Не нравится, что-то неправильно

Lelbry committed:
Немного подправил вывод (ещё проверить перед отправкой)

Lelbry committed:
./project убираю

Lelbry committed:
Внесены изменения:

- Обновлён основной шрифт
- Добавлено новое представление и задокументировано в отчете
- К добавленым новым функциям и процедурам составлно описание и тесты (за 09.04.20024)
- Переработан вывод к курсовой работе

MarinaPaley reviewed (changes requested):
1. "СУБД DBeaver, необходимые для фильтрации набора данных. Научиться создавать простые запросы на фильтрацию данных. Подготовить отчет, включая проверки всех скриптов. " Какая СУБД? Только для фильтрации?
2. Точки после номера не ставят 
![Снимок](https://github.com/Lelbry/IOSU-LABS/assets/26978698/2880950e-0899-43cf-95fe-98ec993ee773)

Lelbry committed:
Переформулирована "цель работы" курсовой работы.

Убраны точки после номера в оглавлении разделов

Lelbry committed:
Исправлено оглавление после изменения стилей

MarinaPaley commented:
@Lelbry Вы, если на завтра я Вам назначу экзамен в Тимсе, например, на 11 и дам 1 час, сможете? И как по курсовой поступить?
 Выучите про нормализацию и про транзакции.
Можете мне в ТГ или в Ватсап написать. +7 910 4133853
